modules/drivers/presto/test/metabase/test/data/presto.clj

```diff
@@ -129,15 +132,15 @@
   [_ s]
   (str/lower-case s))
 
-(defmethod tx/aggregate-column-info :presto
-  ([driver ag-type]
-   ((get-method tx/aggregate-column-info ::tx/test-extensions) driver ag-type))
+#_(defmethod tx/aggregate-column-info :presto
+    ([driver ag-type]
+     ((get-method tx/aggregate-column-info ::tx/test-extensions) driver ag-type))
 
-  ([driver ag-type field]
-   (merge
-    ((get-method tx/aggregate-column-info ::tx/test-extensions) driver ag-type field)
-    (when (= ag-type :sum)
-      {:base_type :type/BigInteger}))))
+    ([driver ag-type field]
+     (merge
+      ((get-method tx/aggregate-column-info ::tx/test-extensions) driver ag-type field)
+      (when (= ag-type :sum)
+        {:base_type :type/BigInteger}))))
```

Comment:
Is it worth just removing this entirely?

Reply:
Yes, done.  It lives in `:presto-common` now.